Solutions/Cloud Identity Threat Protection Essentials/Hunting Queries/SuspiciousSignintoPrivilegedAccount.yaml

```diff
@@ -36,21 +36,29 @@ query: |
   | where ResultType == "0"
```

Comment:
| where AuthenticationRequirement == "singleFactorAuthentication"

To

| where AuthenticationRequirement =~ "singleFactorAuthentication"